db-utils.js

```diff
@@ -1,21 +1,45 @@
 "use strict";
 
+require("dotenv").load();
+
 const pg = require("pg");
 
-// Use PG* env vars to configure client.
-const client = new pg.Client();
+function assertTestingEnvironment() {
+  // eslint-disable-next-line no-process-env
+  if (!process.env.TESTING_ENVIRONMENT) {
+    throw new Error("Attempting to run database setup without TESTING_ENVIRONMENT set, exiting.");
+  }
+}
 
 const DBUtils = {
-  async setupDatabase() {
-    // eslint-disable-next-line no-process-env
-    if (!process.env.TESTING_ENVIRONMENT) {
-      throw new Error("Attempting to run database setup without TESTING_ENVIRONMENT set, exiting.");
-    }
+  async setupUsersTable() {
+    assertTestingEnvironment();
 
+    // Use PG* env vars to configure client.
+    const client = new pg.Client();
     try {
       await client.connect();
       await client.query("DROP TABLE IF EXISTS users;");
-      await client.query("CREATE TABLE users ( email varchar(320) UNIQUE );");
+      await client.query("CREATE TABLE users ( email VARCHAR(320) UNIQUE );");
+    } finally {
+      await client.end();
+    }
+  },
+
+  async setupTempUsersTable() {
+    assertTestingEnvironment();
+
+    // Use PG* env vars to configure client.
+    const client = new pg.Client();
+    try {
+      await client.connect();
+      await client.query("DROP TABLE IF EXISTS users_temp;");
+      await client.query("CREATE TABLE users_temp ( \
+                            email VARCHAR(320) UNIQUE, \
+                            token VARCHAR(80), \
+                            time_added TIMESTAMP DEFAULT NOW() \
+                          );");
```

Comment:
I think if you use template strings with a backtick (``` ` ```), you can avoid doing the multiline backslash trick.

```js
      await client.query(`CREATE TABLE users_temp (
                            email VARCHAR(320) UNIQUE,
                            token VARCHAR(80),
                            time_added TIMESTAMP DEFAULT NOW()
                          );`);
```